resources/views/dashboard/settings/email-setup.blade.php

```diff
@@ -0,0 +1,128 @@
+@extends('layout.dashboard')
+
+@section('content')
+<div class="content-panel">
+    @if(isset($sub_menu))
+    @include('dashboard.partials.sub-sidebar')
+    @endif
+    <div class="content-wrapper">
+        <div class="header sub-header" id="application-setup">
+            <span class="uppercase">
+                {{ trans('dashboard.settings.email.email') }}
+            </span>
+        </div>
+        <div class="row">
+            <div class="col-sm-12">
+                <form id="settings-form" name="SettingsForm" class="form-vertical" role="form" action="{{ route('dashboard.settings.email-post') }}" method="POST" enctype="multipart/form-data">
+                    <input type="hidden" name="_token" value="{{ csrf_token() }}">
+                    @include('dashboard.partials.errors')
+                    <fieldset>
+                        <div class="row">
+                            <div class="col-xs-12">
+                                <div class="form-group">
+                                    <label>{{ trans('forms.setup.mail_driver') }}</label>
+                                    <select name="app_locale" class="form-control" required>
+                                        @foreach($mail_drivers as $key => $driver)
+                                            <option value="{{ $key }}" @if( config('mail.driver') === $key) selected @endif>
```

Comment:
Can you pass this through as a variable (see `SiteComposer`) rather than using the `config` helper.